docs/02.API-REFERENCE.md

```diff
@@ -523,6 +544,40 @@ main (void)
 - [jerry_cleanup](#jerry_cleanup)
 
 
+## jerry_get_memory_stats
+
+**Summary**
+
+Get heap memory stats.
+
+**Prototype**
+
+```c
+bool
+jerry_get_memory_stats (jerry_heap_stats_t *out_stats_p);
+```
+
+- `out_stats_p` - out parameter, that gives the stats of the heap.
+- return value
+  - true, if run was successful
+  - false, otherwise. Usually it is because the MEM_STATS feature is not enabled.
+
+**Example**
+
+```c
+jerry_init (JERRY_INIT_EMPTY | JERRY_INIT_MEM_STATS);
```

Comment:
in jmem-allocator.c
```c
void
jmem_finalize (void)
{
  jmem_pools_finalize ();

#ifdef JMEM_STATS
  if (JERRY_CONTEXT (jerry_init_flags) & ECMA_INIT_MEM_STATS)
  {
    jmem_stats_print ();
  }
#endif /* JMEM_STATS */

  jmem_heap_finalize ();
} /* jmem_finalize */
```
I think the only usage of "JERRY_INIT_MEM_STATS" is dumping the mem stats after cleaning up. So you're right, we don't need it here, i will remove it.